server/Cargo.toml

```diff
@@ -10,8 +10,8 @@ version = "0.1.3"
 build = "build.rs"
 
 [features]
-default = []
-with_ssl = ["openssl", "actix-web/alpn"]
+default = ["with_ssl"]
+with_ssl = ["native-tls", "actix-web/tls"]
```

Comment:
Can we please call this feature `"ssl"`? See https://rust-lang-nursery.github.io/api-guidelines/naming.html#c-feature

Reply:
done, forgot to psuh